### PR TITLE
Enable total column type to be set as an Integer or Float

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer 

This allows us to specify the total column type.

### CODE REVIEW

File: wazimap/data/tables.py [hunk 1]
```diff
@@ -340,6 +348,7 @@ def __init__(self, fields, id=None, universe='Population', description=None, den
         self.denominator_key = denominator_key
         self.table_per_level = table_per_level
         self.has_total = has_total
+        self.total_type = TOTAL_COL_TYPES[total_type]
```
Comment (longhotsummer) [inline wazimap/data/tables.py]: I suggest this instead, which is more flexible:

```
import sqlalchemy.types
...

self.total_type = getattr(sqlalchemy.types, total_type)
```

File: wazimap/data/tables.py [hunk 2]
```diff
@@ -310,7 +317,7 @@ class FieldTable(SimpleTable):
 
     """
     def __init__(self, fields, id=None, universe='Population', description=None, denominator_key=None,
-                 table_per_level=False, has_total=True, **kwargs):
+                 table_per_level=False, has_total=True, total_type='Integer', **kwargs):
```
Comment (longhotsummer) [inline wazimap/data/tables.py]: Can we make this `value_type` throughout instead? Linking it to the total seems weird for tables for which we can specify `has_total=False`.

**Review by longhotsummer — CHANGES_REQUESTED** [review]

File: wazimap/data/tables.py (hunk 1 above)
Comment (xybrnet) [reply wazimap/data/tables.py]: That is much better. I have changed it to that.

File: wazimap/data/tables.py (hunk 2 above)
Comment (xybrnet) [reply wazimap/data/tables.py]: Cool, that makes more sense. Have done so.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**